src/plugins/intel_gpu/src/plugin/ops/convolution.cpp

```diff
@@ -137,11 +137,12 @@ static void CreateConvolutionBackpropDataOp(Program& p, const std::shared_ptr<ng
 
     auto weightsName = inputs[1];
     auto weights_node = op->get_input_node_shared_ptr(1);
-    // WA: For the cases like Const(weights)->Sub(zp)->Deconv.
+    bool nodeOnConstPath = IsNodeOnConstPath(weights_node);
```

Comment:
It's more about weights node, so wouldn't it be better to call the variable something like `HasConstantWeights` or something else related to weights node

Reply:
I suppose the variable can be renamed as you suggest, thanks.